beacon-chain/rpc/eth/light-client/handlers.go

```diff
@@ -67,10 +67,22 @@ func (s *Server) GetLightClientBootstrap(w http.ResponseWriter, req *http.Reques
 		httputil.WriteJson(w, response)
 		return
 	case version.Bellatrix:
```

Comment:
you can merge the two cases by writing `case version.Altair, version.Bellatrix:`